price_rule.go

```diff
@@ -84,6 +84,31 @@ type PriceRulesResource struct {
 	PriceRules []PriceRule `json:"price_rules"`
 }
 
+func (pr *PriceRule) SetPrerequisiteSubtotalRange(greaterThanOrEqualTo string) {
```

Comment:
based on the nullability of these fields in the model, the setters should also accommodate nil